localization/english/dvg_flavor_bohemia_l_english.yml

```
@@ -26,4 +26,4 @@
 
  HRE_success_tt: "On completion of both Journal entries, the ability to form the nation of the Holy Roman Empire will unlock."
  hre_state_tt: "Own or have a subject own all the state regions of the Holy Roman Empire formable. This can be checked from the nations formables menu."
- hre_variable_tt: "Has completed both Journal entries succesfully"
\ No newline at end of file
+ hre_variable_tt: "Has completed both Journal entries succesfully"
```

```suggestion
 hre_variable_tt: "Has completed both Journal entries successfully"
```
Noticed this a second before you merged it.